tqdm/tests/tests_tqdm.py

```diff
@@ -116,8 +116,11 @@ def write(self, s):
         self.text += s
         self.cursor = len(self.text)
```

Comment:
Sorry I'm stupid, I forgot I could change the branch. I will apply the edits myself, don't worry.